url/src/comp.php

```diff
@@ -220,156 +225,308 @@ function counter_summary() {
 }
 
 function get_resource_id($resource_data) {
-	global $myfile;
-
-	if (! empty( $resource_data->included[0]->id )) {
-		$resource = $resource_data->included[0]->id;
+	if (! empty( $resource_data->id )) {
+		$resource = $resource_data->id;
 		echo_message_to_screen(DEBUG, "Resource ID: $resource \t");
-		fwrite($myfile,$resource . ",");
-		return $resource;	
+		return $resource;
 	} 
-	fwrite($myfile,",");
-	return false;
+	return "";
 }
 
-function get_webaddress_array($resource_data) {
-	global $myfile;
-
-	if (! empty( $resource_data->included[0]->attributes->web_addresses )) {
-		$web_addresses = $resource_data->included[0]->attributes->web_addresses;		
+function get_web_address_array($resource_data) {
```

Comment:
Renamed this for no reason other than the 'spelling' error in my IDE annoyed me ;)